# Published
Figures I created using R (at least in part, touched up manually) are compiled here. Publications in order of release date.

#
#

# Universal activity-based labeling method for ammonia- and alkane-oxidizing bacteria

(https://www.nature.com/articles/s41396-021-01144-0#Sec14)

**ISME fig 8**

![ISME fig 8](isme_2021/isme8.png)

Normalized read abundances of **a**, **b** CuMMO genes and **c**, **d** _rpoB_ in the nitrifying enrichment culture of untreated biomass, controls, and sorted biomass for the activity-based labeling protocol (see Supplementary Table S2 for details). Gene abundances were calculated by normalizing paired read counts by gene length and million reads mapped to the co-assembly. The total number of reads mapped to the co-assembly are indicated as gray bars scaled to fit in the same range as the normalized abundances. Colors denote the taxonomic affiliation of the marker genes. The same genes for the taxa of interest (_Nitrosomonas_, comammox _Nitrospira_, and putative alkane oxidizers) are emphasized and connected with lines. **b**, **d** show the same data as (**a**, **c**) but as proportion of the total normalized read abundance for the respective gene in the metagenomic sample, with each individual colored box representing the relative abundance of a single gene from the co-assembly. The final treatment preceding the CuAAC reaction, the CuAAC controls for the influence of 1,7OD inactivation and dye addition, and the final sorted sample are indicated in bold.

**ISME fig 9**

![ISME fig 9](isme_2021/isme9.png)

Contig abundances were calculated by normalizing paired read counts by gene length and million reads mapped to the assembly. Each circle represents an assembled contig ≥1500 bps, indicated by circle size. Contigs assigned to the automated bins of taxa of interest are highlighted using specific colors. Contigs with genes encoding members of the CuMMO protein family are emphasized with fill circles and text. For added clarity, dashed lines were drawn with a slope of 1 and 50 to guide estimations of the differences in normalized abundances between the samples. The plot is zoomed in on the region containing >80% of all data, the full version is provided as Supplementary Fig. S6.

**ISME fig S4**
![ISME fig s4](isme_2021/ismeS2.png)

**Figure S4.** Unrooted phylogenetic reconstruction of full-length CuMMO gene references (Pfam PF02461) and genes recovered from three metagenomic assemblies. Circles indicate nodes with statistical support, branch colors highlight clades of interest, and triangles denote the assembled genes while reference clades are unlabeled. The inset shows the outlined region of the tree to clarify the topology of this dense region, with the accession numbers of the reference sequences displayed. _amoA_, ammonia monooxygenase subunit A; _hmoA_, alkane monooxygenase subunit A; NB, nitrifying bioreactor; WWTP, wastewater treatment plant. The scale bar represents 0.3 amino acid substitutions per alignment position.

**ISME fig S5**

![ISME fig s5](isme_2021/ismeS3.png)

**Figure S5.** Linear correlations of the CuMMO gene and _rpoB_ normalized read abundances of the dominant ammonia- and putative alkane-oxidizing bacteria in the nitrifying enrichment culture. Each symbol represents the Reads Per Kilobase gene length per Million mapped reads (RPKM) of a gene in one metagenomic sample, colored by taxonomic affiliation to match Figure 7. The linear relationship and the 95% confidence interval are shown as a line and background of matching color and are labelled with the coefficient and the slope of the line in matching color.

#
#

# Microbial Contributions to Carbon, Nitrogen, and Greenhouse Gas Cycling in Freshwater Terrestrial-Aquatic Interfaces

(https://www.proquest.com/intermediateredirectforezproxy)

  *Images in the Thesis folder only included if not another folder, those labelled with a "x" have components or data currently under review

**Thesis fig 2**

![Thesis fig 2](thesis_2020/thesis2.jpg)

**Figure 2. Distribution of United States watersheds.** A Mercator projection of the United States (dark grey) overlaid with rivers and the watershed networks (light blue) to display the extent and spatial patterns of the exposed and subsurface freshwater terrestrial-aquatic interfaces.

**Thesis fig 17**

![Thesis fig 17](thesis_2020/thesis17x.png)

**Figure 17. Experimental overview of hyporheic zone (HZ) sampling in the riparian zone of the Columbia River.** (A) shows the geographic location in Washington (USA) of the sampling site. (B) is an aerial view of the site to show the two transects and difference in vegetation between them. (C) is a schematic drawing of the sampling design in which green indicates the vegetated (“Veg.”) transect and brown indicates the non-vegetated (“Non-Veg.”) transects, and the sediment profiles were sub-sectioned into 10 cm intervals. (D) outlines the microbial community (“Comm.”) and porewater chemistry (“Chem.”) data collected for these analyses. The datasets included metagenomes (“MetaG”), metaproteomes (“MetaP”), and metabolomes (“MetaB”), as well as measurements of total carbon (“Tot. C”), total nitrogen (“Tot. N”), and ammonium (“NH4+”).

**Thesis fig 18**

![Thesis fig 18](thesis_2020/thesis18x.png)

**Figure 18. Butterfly plot of the detection frequency of organic metabolites identified using 1H Nuclear Magnetic Resonance (NMR)**. Over 29 metabolites were detected above minimum concentration thresholds (1 μM prior to correction) in at least one sample. The number of samples each of these metabolites were detected in is split between the non-vegetated and vegetated transects. Within substrate categories, the metabolites are in increasing rank order by number of total samples from top to bottom.

**Thesis fig 19**

![Thesis fig 19](thesis_2020/thesis19x.png)

**Figure 19. Sediment porewater chemistry profiles at the two transects.** Mean and standard errors for ammonium (A), resazurin as a proxy for aerobic respiration (B), total nitrogen (C), and total carbon (D) are shown over the 10 cm increments and aggregated between the two transects. Previous biochemical analysis identified “low activity” and “hotspots” of microbial activity based on the resazurin reduction assay.

**Thesis fig 20**

![Thesis fig 20](thesis_2020/thesis20x.png)

**Figure 20. Metagenome-Assembled Genome (MAG) phylogenetic distribution and activity in HZ sediments.** (A) Count of recovered MAGs (55 total) colored by their phylum and stacked according to the first empty position within the taxonomic string provided by the Genome Taxonomy Database toolkit (GTDB) as a proxy for genomic novelty 145. (B) Mean relative abundance and standard error of peptide recruitment to each MAG, colored by phylum. (C) Balloon chart of mean peptide relative abundances for each MAG, grouped and colored by phylum, at the 10 cm sediment profile intervals. For all sections, the colors of each phylum match the legend inset into (B).

**Thesis fig 22**

![Thesis fig 22](thesis_2020/thesis22x.png)

**Figure 22. Summary of shared metaproteomic content between transects.** Euler diagram of the proportional to the number of total proteins recruiting peptides in each transect, and the overlap represents the proportion of these proteins recruiting peptides in both transects. The proportion of shared proteins recruiting peptides between depths (0-30 cm and 40-60 cm) was similar to the shared content between the transects. For this analysis, only proteins recruiting 2 or more total peptides were included (n=898) in order to allow for recruitment of at least 1 peptide for a given protein to both transects or both depths and reduce false positives.

**Thesis fig 26**

![Thesis fig 26](thesis_2020/thesis26x.png)

**Figure 26. Nitrate reducing repertoire of HZ sediment MAGs.** Maximum likelihood phylogeny of dimethyl sulfoxide family reductases (Uniprot PF00384), reconstructed using FastTree with default parameters on an alignment of 4,062 sequences and 8,313 residues trimmed to remove end gaps generated using MAFFT. Sequences recovered from MAGs assigned as periplasmic nitrate reductase (_nap_), respiratory nitrate reductase (_nar_), and nitrite oxidoreductase (_nxr_) are indicated at their tips. The _nxr_-like nar group includes sequences that share high sequence homology with nxr but current evidence suggest they perform the same biochemical reactions as _nar_.

**Thesis fig 27**

![Thesis fig 27](thesis_2020/thesis27x.png)

**Figure 27. Nitric oxide reductase (_nos_) recovery in HZ sediment MAGs.** Maximum likelihood phylogeny of nos protein cluster retrieved from NCBI, reconstructed using FastTree with default parameters on an alignment of 304 sequences and 620 residues, trimmed to remove end gaps generated using MAFFT. Sequences recovered from MAGs assigned to nos are indicated as squares on their tips. The physiological differences between Clade I and Clade II nos are detailed in Suenaga et al. and Toon et al..

**Thesis fig 32**

![Thesis fig 32](thesis_2020/thesis32.png)

**Figure 32. Distribution of rpS3 sequences identified in freshwater Terrestrial-Aquatic Interface (TAI) habitats.** Phylogenetic composition of rpS3 sequences recovered from the subset of environments that can be transiently or permanently classified as a TAI are shown as circles grouped and colored by methanotrophic lineage. Numbers in the legend indicate the total number of rpS3 sequences assigned to the lineage per information type, i.e. whether the sequence was retrieved from an isolate genome, a MAG, a contiguous sequence (contig), or a gene alone. The area of the circle (log10-scaled) indicates the number of rpS3 sequences per lineage and freshwater TAI habitat, with single representatives depicted as a not-to-scale black dot with colored outline to improve visibility. The relative variability indicates the consistency of an ecosystem to all within the category of freshwater TAI, which can be due to seasonality (“Temporal”), land cover and subsurface heterogeneity (“Spatial”), or relatively consistent but still subject to dynamic processes (“Stable”).

**Thesis fig 37**

![Thesis fig 37](thesis_2020/thesis37.png)

**Figure 37. Water table dynamics in selected North American freshwater Terrestrial-Aquatic Interface (TAI) sites.** Yearly dynamics of gage height (a proxy for water depth) are shown since 2009 (for visibility over the full datasets) and colored by year from green (least recent) to brown (most recent). A brief description of change since the initiation of data collection is summarized in the upper left corner, and the gray boxes represent noticeable yearly trends within the time periods shown. All data were retrieved in October 2019 using United States Geological Society National Water Information System web interface: https://waterdata.usgs.gov/nwis/rt .

#
#

# Metagenomic Approaches Unearth Methanotroph Phylogenetic and Metabolic Diversity 

(https://www.caister.com/cimb/v/v33/57.pdf)

 -data and code originally published here: https://github.com/TheWrightonLab/Methanotroph_rpS3Analyses_SmithWrighton2018
 
 *Images with an "u" have been slighlty modified to match the version in my thesis

**CIMB fig 3**

![CIMB fig 3](cimb_2019/cmib3u.png)

**Figure 3.3** Phylogeny of methanotroph ribosomal protein S3 (rpS3) genes identifed in metagenomic datasets and isolates. A total 566 recovered and 152 reference – 77 isolates and 75 Metagenome-Assembled Genomes (MAGs) – rpS3 genes were used for analysis if the rpS3 gene was greater than 180 amino acids, approximately 50% of the maximum reference sequence length. Protein sequences were aligned using MUSCLE and modifed to remove predominantly gapped, clade-specifc positions resulting in 183 residues. The maximumlikelihood phylogenetic tree was generated using RAxML with the GAMMA model of rate heterogeneity and WAG substitution matrix, with 100 bootstrap replicates. Auxiliary layers around the tree are colour-coded for metadata accompanying the genomic information and indicate cultivation status (Sample Treatment), the type of sequencing performed (Sequencing Type), and the environmental source (Ecosystem) of the samples and genomic information. No information is provided for non-methanotrophic out-group sequences.

**CIMB fig 4**

![CIMB fig 4](cimb_2019/cmib4.png)

**Figure 3.4** Phylogenetic composition of rpS3 genes recovered in putative methanotrophic MetagenomeAssembled Genomes (MAGs). Distribution of MAGs at the genus level, if known, among the major clades of methanotrophs and grouped by family according to SILVA (SSU refNR 132) and the Genome Taxonomy DataBase (GTDB). Yellow stars indicate the lack of axenic, i.e. pure, cultures of specifc methanotrophic genera (Proteobacteria and Verrucomicrobia) and classes (Methylomirabilota and Methanomicrobia).

**CIMB fig 5**

![CIMB fig 5](cimb_2019/cmib5u.png)
**
Figure 3.5** Geographic distribution of methanotrophs detected in metagenomic and metatranscriptomic datasets across the globe. Each point represents one of 327 unique BioProjects associated with a metagenome containing a methanotroph rpS3 gene or a putative methanotrophic Metagenome-Assembled Genome (MAG). Phylogenetic afliation of rpS3 genes not in MAGs were assigned to the class of the most similar reference sequence determined by BLASTp and confrmed by topological coherence with reference genes. The colours indicate the phylogenetic afliation of the rpS3 gene, and the size of the point represents the number of recovered sequences per unique BioProject. Sixteen MAGs lacked identifable location information.

#
#

# Members of the Genus Methylobacter Are Inferred To Account for the Majority of Aerobic Methane Oxidation in Oxic Soils from a Freshwater Wetland

(https://journals.asm.org/doi/10.1128/mBio.00815-18)
 
**mBio fig 1**

![mBio fig 1](mbio_2018/mbio1.png)

**FIG 1** Overview of OWC field site and methane dynamics. (A) Old Woman Creek (OWC) National Estuarine Research Reserve is a 571-acre, NOAA-operated temperate freshwater wetland near Lake Erie in Ohio. Soils were sampled from an ecological transect composed of the following land cover types: _Typha_ vegetated (Plant, green), periodically flooded mud flat (Mud, orange), and continually saturated water channel (Water, blue). We selected two soil depths as representative oxic and anoxic soil zones (0 to 5 cm, Surface; 23 to 35 cm, Deep). (B) Soil _in situ_ methane concentration variation by depth and land cover type over the four sampled seasons (November 2014 through August 2015). Different months are represented as different shades of gray or are colored by land cover and depth to match the curves shown in panel C. (C) Aerobic methane consumption potential curves of surface and deep soil incubations. Points and curves are colored by the land cover type and depth in the soil column, matching the samples highlighted in panel B.

**mBio fig 2**

![mBio fig 2](mbio_2018/mbio2.png)

**FIG 2** Relative abundances of dominant methanotrophic taxa. (A) Stacked bar chart of the 10 most abundant microbial orders in all soil samples (n  66). The total number of OTUs in each order is noted above the stacked bar chart, and the relative ranks of the 20 most abundant OTUs are indicated. Four dominant _Methylococcales_ OTUs are highlighted with shades of red. The inset shows fold enrichment of the dominant OTUs in surface soils over deep soils, with significant differences (analysis of variance with Tukey’s range test; adjusted _P_ value [_P_-adj], <0.05) indicated by asterisks. (B) The abundances of the four dominant OTUs compared to those of other detected methane-oxidizing taxa. Shown are the 10 next most abundant _Methylococcales_ taxa and the most abundant OTUs of other methanotroph taxa. Dots represent individual samples, and the black bar represents the average. The four dominant OTUs were significantly more abundant (analysis of variance with Tukey’s range test; _P_-adj, <0.05) than all of the other putative methanotrophic OTUs.

**mBio fig 3**

![mBio fig 3](mbio_2018/mbio3.png)

**FIG 3** Phylogenetic placement of novel genomes and overview of methane and dissimilatory nitrate reduction metabolisms. (A) Maximum likelihood tree constructed from 21 concatenated universally conserved single-copy and ribosomal protein genes (3,270 amino acids [aa]), rooted to _Nitrosococcus_. Our four genomes represent two novel lineages (Text S1): OWC _Methylobacter_ (red) and NSP1-2 (black). Ecosystem types are indicated by colored circles next to the genome name. (B) Inventory of the presence and types of particulate methane monooxygenase (_pmo_), _pmo_-like (_pxm_), and methanol dehydrogenase (_mxaF_, _xoxF_) genes in _Methylococcales_ genomes (Data Set S1). (C) Dissimilatory nitrate and nitrite reduction marker genes found in _Methylococcales_ genomes. Gene expression or biochemical transformations demonstrating dissimilatory nitrate or nitrite reduction are indicated by the asterisks (Data Set S1). Inventories are not shown for NSO1-1 due to lack of core genes.

**mBio fig 4**

![mBio fig 4](mbio_2018/mbio4.png)

**FIG 4** OWC _Methylobacter_ and NSP1-2 gene expression in surface soils. (A) Rank abundance curve of the top 100 annotated genes by average normalized gene expression (FPKM [fragments per kilobase of exon per million mapped reads]) in surface soils (n  12). The approximate positions of the top 1% and 3% of the 22,219 genes with detectable transcripts are indicated. (B) Box plots of the mean expression levels of representative genes from core methane (Gen, generation; Ox, oxidation), carbon (RuMP/PPP, ribulose monophosphate pathway or pentose phosphate pathway), and energy generation (C1, C1-transfer) pathways and bulk ribosomal protein gene transcript abundances. The data from the genes assigned to OWC _Methylobacter_ and NSP1-2 and the _mcr_ gene of “_Ca._ Methanothrix paradoxum” were averaged across all 12 samples and overlaid onto the box plots (colored circles; see Data Set S1). Abbreviations are as follows: _mcr_, methyl coenzyme-M reductase; _pmo_, particulate methane monooxygenase; _mdh_, methanol dehydrogenase; _hps_, hexulose-phosphate synthase; _hpi_, hexulose phosphate isomerase; _tkt_, transketolase; _tal_, transladolase; _fba_, fructose 1,6-bisphosphate aldolase; _tpi_, triose phosphate isomerase; gapdh, glyceraldehyde phosphate dehydrogenase; _fae_, formaldehyde-activating enzyme; _fdh_, formate dehydrogenase; _nuo_, NADH dehydrogenase; _nqr_, Na(+)-translocating NADH:ubiquinone oxidoreductase; _pet_, ubiquinol cytochrome bc reductase; Cyt c Ox, cytochrome _c_ oxidase; _nar_, respiratory nitrate reductase; _nirK_, copper-containing nitrite reductase; _nirS_, cytochrome _cd1_ nitrite reductase; _nor_, nitric oxide reductase; _rps_, small subunit ribosomal protein; _rpl_, large subunit ribosomal protein.

**mBio fig 5**

![mBio fig 5](mbio_2018/mbio5.png)

**FIG 5** Distribution of OWC _Methylobacter_ _pmoA_ genes detected in publicly available sequencing databases. Numbers indicating specific locations are indicated in Data Set S1, and the shading indicates the type of sequencing performed in the study as follows: Active _in situ_, detected in environmental metatranscriptomes; Present _in situ_, detected in environmental metagenomes or clone libraries; Enriched, detected in incubation sequencing studies. NSP1-2 was found at the locations numbered 1, 2, 5, and 7. The cyan star indicates the bubble representing this study. A representation of the sequences assigned to OWC _Methylobacter_ and NSP1-2 is visualized in Fig. S7, and the full list of accession numbers and the accompanying metadata are available in Data Set S1.

**mBio fig S1**

![mBio fig S1](mbio_2018/mbioS1.png)

**Figure S1.** Dominant Methylococcales OTUs abundance patterns along spatial, temporal, and geochemical gradients. (A) Fold differences of the OTUs between seasons (left) and sites (right), with significant differences (analysis of variance with Tukey’s range test, p-adj<0.05) indicated by asterisks. (B) Significant correlations (P<0.05) between dominant methanotroph OTUs and methane consumption rates (top), dissolved oxygen concentrations (bottom). The linear relationships are shown only for significant correlations.

**mBio fig S6**

![mBio fig S6](mbio_2018/mbioS6.png)

**Figure S6.** Expression of _pmoA_ in November and August surface soils (n=12). (A) Stacked bar chart showing the relative contributions of OWC _Methylobacter_ and NSP1-2 genotypes to the total _pmoA_ detected in the metatranscriptomes, averaged across seasons and land covers. (B) Normalized transcript abundances (FPKM, Fragments Per Kilobase exon per Million reads) of _pmoA_ genes recovered in OWC _Methylobacter_ and NSP1-2 genomes, averaged across seasons and land covers. (C) Normalized transcript abundances (FPKM) of _mcrA_ genes recovered in _Candidatus_ Methanothrix paradoxum genomes, averaged across seasons and land covers. Insets show the reduction in activity between seasons, and asterisks denote a significant change between seasons (analysis of variance with Tukey’s range test, p-adj<0.05).

**mBio fig S7**

![mBio fig S7](mbio_2018/mbioS7.png)

**Figure S7.** Simplified depiction of OWC _Methylobacter_ and NSP1-2 _pmoA_ genes identified in publicly available environmental sequencing studies. Nucleotide reference tree of Methylococcales _pmoA_ rooted to _Nitrosococcus_ _amoA_ (not shown), and public sequence data placements. Metadata provided for each Biosample or Popset where sequences similar to OWC _Methylobacter_ or NSP1-2 were detected are summarized in the grid below. Analyses were not performed on cultivated Methylococcales strains and the metadata is not reported for the reference or unassigned sequences. Confidence indicates an arbitrary strength of the affiliation with our genomes: H, denotes high confidence placements onto a genome in at least one of the phylogenies and on the genome or its immediate ancestor (only OWC _Methylobacter_) in the other phylogeny; M, refers to medium confidence by placement on a genome in either nucleic acid or amino acid phylogenies. Sequencing refers to the type of sequencing data: T, metatranscriptomic; G, metagenomic; C, clone library. Represented Hits describes the total number of sequences represented by the placement in this figure. Incubation indicates treatment of samples: N, none; I, incubation study. Freshwater describes the freshwater source of the samples: G, groundwater; L, lake; W, wetland. Soil describes the soil type: A, agricultural; P, permafrost; D, sediment; S, soil.
